internal/xpkg/v2/testdata/config_package.yaml

```diff
@@ -0,0 +1,313 @@
+apiVersion: apiextensions.crossplane.io/v1
```

Comment:
I think this test file is from a very long time ago and isn't actually used in tests anymore. Over in up, if I remove it and run `make test` the tests still pass.

Reply:
Same with the `config_meta.yaml` file above.